app/models/supplejack_api/record.rb

```diff
@@ -6,6 +6,119 @@ class Record
     include Support::Searchable
     include Support::Harvestable
     include Support::FragmentHelpers
-    include SupplejackApi::Concerns::Record
+
+    attr_accessor :next_record, :previous_record, :next_page,
+                  :previous_page, :should_index_flag
+
+    # Associations
+    embeds_many :fragments, cascade_callbacks: true, class_name: 'SupplejackApi::ApiRecord::RecordFragment'
+    embeds_one :merged_fragment, cascade_callbacks: true, class_name: 'SupplejackApi::ApiRecord::RecordFragment'
+
+    # rubocop:disable  Rails/HasAndBelongsToMany
+    has_and_belongs_to_many :concepts, class_name: 'SupplejackApi::Concept'
+    # rubocop:enable  Rails/HasAndBelongsToMany
+
+    # From storable
+    store_in collection: 'records'
+    index(concept_ids: 1)
+    index({ record_id: 1 }, unique: true)
+
+    auto_increment :record_id, client: 'strong'
+
+    # Callbacks
+    before_save :merge_fragments
+    after_save :remove_from_index
+
+    # Scopes
+    scope :active,          -> { where(status: 'active') }
+    scope :deleted,         -> { where(status: 'deleted') }
+    scope :suppressed,      -> { where(status: 'suppressed') }
+    scope :solr_rejected,   -> { where(status: 'solr_rejected') }
+
+    build_model_fields
+
+    def self.created_on(date)
+      where(:created_at.gte => date.at_beginning_of_day, :created_at.lte => date.at_end_of_day)
+    end
+
+    def self.find_multiple(ids)
+      return [] unless ids.try(:any?)
+      string_ids = ids.find_all { |id| id.to_s.length > 10 }
```

Comment:
Could 10 and 100 below be stored as constant values?

Reply:
stored them as vars for now